gossip/src/crds.rs

```diff
@@ -669,6 +690,18 @@ impl CrdsDataStats {
                 self.votes.put(slot, num_nodes + 1);
             }
         }
+
+        // check trailing zero bits on signature for metrics
+        if trailing_n_crds_signature_bits_are_zero(
+            entry.value.signature.as_ref(),
+            SIGNATURE_SAMPLE_TRAILING_ZEROS,
+        ) {
+            datapoint_info!(
+                "cluster_info_crds_message_signatures",
+                ("crds_origin", entry.value.pubkey().to_string(), String),
+                ("crds_signature", entry.value.signature.to_string(), String)
```

Comment:
This is 32 + 64 bytes for pubkey + signature.
To reduce load on metrics sever, I am wondering if it makes sense to trim these to smaller number of bytes. Most likely they will be still unique distinguishable.

Reply:
ya this is a good point. since we're already filtering on trailing zeros. So say we only report the first 16 bytes of the signature, the chance that the last 18 bits and the first 128 bits (16 * 8) are the same will be very low. Could even just take first 8 bytes of signature. change of collision: 1/2^(18+64) = 1/4.8x10^24